std/http/file_server.ts

```diff
@@ -142,20 +142,26 @@ async function serveDir(
 }
 
 async function serveFallback(req: ServerRequest, e: Error): Promise<Response> {
-  if (
-    e instanceof Deno.DenoError &&
-    (e as Deno.DenoError<Deno.ErrorKind.NotFound>).kind === ErrorKind.NotFound
-  ) {
-    return {
-      status: 404,
-      body: encoder.encode("Not found")
-    };
-  } else {
-    return {
-      status: 500,
-      body: encoder.encode("Internal server error")
-    };
+  if (e instanceof DenoError) {
+    if (e.kind === ErrorKind.PermissionDenied) {
+      return {
+        status: 500,
+        body: encoder.encode(e.message)
```

Comment:
Rather than serve the message, you should log the error to the console near the `serveFallback()` call in `listenAndServe()`. That's what was suggested by @kitsonk.

Reply:
hmm...so I have reverted these commits